pkg/metrics/metrics.go

```diff
@@ -0,0 +1,186 @@
+package metrics
+
+import (
+	"strconv"
+
+	"github.com/golang/glog"
+	mapiv1beta1 "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
+	machineinformers "github.com/openshift/cluster-api/pkg/client/informers_generated/externalversions/machine/v1beta1"
+	machinelisters "github.com/openshift/cluster-api/pkg/client/listers_generated/machine/v1beta1"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/utils/pointer"
+)
+
+var (
+	// MachineCountDesc is a metric about machine object count in the cluster
+	MachineCountDesc = prometheus.NewDesc("mapi_machine_items_total", "Count of machine objects currently at the apiserver", nil, nil)
```

Comment:
It looks like a gauge so shouldn't have with a `_total` suffix which are reserved for counters by convention.